docs/how_to_guides/index.rst

```diff
@@ -8,6 +8,9 @@ How To Guides
    how_to_setup_RO_config_options
    how_to_setup_simple_chemistry
    how_to_use_apparent_and_true_chemical_species
+   how_to_use_inherent_reactions
+   how_to_scale_chemical_reactions
+   how_to_scale_chemical_species
    how_to_scale_a_model
```

Comment:
I think it would make more sense for `how_to_scale_a_model` to come before any of the other scaling docs because it gives a general intro to scaling in general. 

However, if you look at the last section of `how_to_scale_a_model`, it recommends using `user-scaling`, which was meant to be targeted toward the RO model. So we should probably specify that since the chemistry scaling docs recommend not using `user-scaling`. This contradiction between the two could confuse readers.

Reply:
At this moment, I am not sure we can do much about this contradiction. The chemistry modules generally don't solve with user-scaling and the RO modules generally don't solve with gradient-based scaling. Once this issue is ultimately resolved, after we get a working flowsheet, then we can update all documentation to reflect those changes. 

Also, it may make more sense for <code>how_to_scale_a_model</code> to be changed to <code>how_to_scale_the_RO_model</code> since the current title implies it is universally applicable.

Reply:
I totally agree with regard to the contradiction and resolving the discrepancy between scaling chem and RO modules. I think we can provide a temporary remedy by mentioning something like the following in your initial warning about user-scaling: "While "user-scaling" is currently the most stable option for unit models such as RO, we do not recommend using this option for chemistry modules at this time." 

Another note: that last section in `how_to_scale_a_model` is actually outdated in the way that it calls for the solver, so it seems that little section will need updating (not for this PR). I think we should keep the generic title of 'how_to_scale_a_model' though.